libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -18,6 +18,17 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
 
     // 3 was for ACCEL_RP_MAX
 
+#if APM_BUILD_TYPE(APM_BUILD_ArduSub)
+    // @Param: ACCEL_Y_MAX
```

Comment:
can you explain why this change is needed? Is it just to change the default value (which could be done with just changing a define, or in the ArduSub directory with an override)?
Or is it the comments?